Library/Formula/v8.rb

```diff
@@ -21,6 +21,7 @@ class V8 < Formula
 
   depends_on :python => :build # gyp doesn't run under 2.6 or lower
   depends_on "readline" => :optional
+  depends_on "icu4c" => :recommended
```

Comment:
And is warning "String '/usr/local/opt' still exists in these files:" (see https://travis-ci.org/Homebrew/homebrew/jobs/104209843#L933) matters? I guess it is non-avoidable when ICU used, but anyway, I don't like such yelling. May it lead to some problems under homebrew infrastructure?